### ISSUE TITLE
Failure to Communicate is an appropriate name for this broken quest

### ISSUE THREAD

**Issue by Ripper1302** [Starfield-Community-Patch/Starfield-Community-Patch#326]
Alban Lopez will not speak to me to move this quest forward.  He is standing outside his home on Guniibucu I-A but will not engage in any conversation so I can not more forward to repair the satellites.   I have been to his farm multiple times over several weeks of game time and there is no change, except the last time the doors on his home were both floating in the air.

I have done file integrity checks at least twice.

**Comment by Constellation-VASCO** [comment]
I just completed this entire questline a couple days ago without issue.
I really wish they'd come up with a way to Reset Quests or at least Cancel them

**Comment by everseeking** [comment]
Having the same issue - unable to trigger dialogue with Lopez at his farm ("E" to talk is shown, and shown being pressed but nothing happens).

I also managed to stumble upon the space station that is part of the quest - I cleared the station thinking I could then get Lopez to trigger a conversation but that did not work either. He appeared both at that station and back on the planet - quest is just stuck.
